build/gen_stub.php

```diff
@@ -169,7 +169,7 @@ public static function fromNode(Node $node) {
 
     public function isNullable(): bool {
         foreach ($this->types as $type) {
-            if ($type->isNull()) {
+            if ($type->isNull() || $type->name === "mixed") {
```

Comment:
Any better way to make `mixed` nullable?

Reply:
Is this needed for the "null default" check? It seems a bit weird to generate IS_MIXED with nullable flag explicitly set.

Reply:
Yes. I somehow thought that we need that flag added in other places. But yeah, `mixed`'s implementation takes care of the nullable case so no need for it, surely.